samples/apps/copilot-chat-app/webapi/Config/NotEmptyOrWhitespaceAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace SemanticKernel.Service.Config;
+
+/// <summary>
+/// If the other property is set to the expected value, then this property is required.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+internal sealed class NotEmptyOrWhitespaceAttribute : ValidationAttribute
+{
+    protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)
+    {
+        if (value == null)
```

Comment:
Shouldn't null be considered like an empty string?

Reply:
The pattern for these validators is that you want them to be able to stack, so "not empty or whitespace" should also allow for something to not be "required" / null.